eclair-core/pom.xml

```diff
@@ -235,6 +235,12 @@
             <version>0.9.8</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
why do we need this?

Reply:
We need this for our docker tests, I've added a comment in the pon